src/Files.App/Files.App.csproj

```diff
@@ -69,9 +69,9 @@
         <PackageReference Include="FluentFTP" Version="43.0.1" />
         <PackageReference Include="ini-parser-netstandard" Version="2.5.2" />
         <PackageReference Include="LibGit2Sharp" Version="0.30.0" />
-        <PackageReference Include="LiteDB" Version="5.0.19" />
+        <PackageReference Include="LiteDB" Version="5.0.20" />
```

Comment:
Can we remove LiteDB? We had it to migrate folder preferences from older versions, but I think most users have already updated.

Reply:
If the time to remove has come. Do you want to remove it in this PR?

Reply:
@hez2010 fyi

Reply:
Could be a separate PR, this just reminded me..

Reply:
The PR for removal will be very simple, so we can make a PR so that we don't forget it, and then merge it when the time has come.

Reply:
I would like to bake for a little bit more time before removing it so that most people can get their installed Files updated.

Reply:
99% of users have updated to the latest release.

Reply:
Seems that you are already removing it. I don't have strong opinion on this so just go for it :)